lib/ex_rabbit_m_q/consumer.ex

```diff
@@ -426,6 +428,22 @@ defmodule ExRabbitMQ.Consumer do
         end
       end
 
+      defp xrmq_exchange_declare(channel, %{exchange_opts: nil}) do
+        :ok # Ignore when not set
+      end
+
+      defp xrmq_exchange_declare(channel, %{exchange_opts: opts}) do
+        AMQP.Exchange.declare(channel, opts[:exchange], opts[:type] || :direct, opts)
+      end
+
+      defp xrmq_queue_bind(channel, queue, %{bind_opts: nil}) do
+        :ok # Ignore when not set
+      end
+
+      defp xrmq_queue_bind(channel, queue, %{bind_opts: opts}) do
+        AMQP.Queue.bind(channel, queue, opts[:exchange], opts[:extra_opts] || [])
```

Comment:
we should break away from the bind_opts.exchange config and use the newer exchange_opts section.
Make sure to update to new minor version